devs/docs/es-backports.rst

```diff
@@ -535,23 +535,23 @@ should be crossed out as well.
 - [ ] f4989c54c45 Revert "serialize initial_connect_timeout as xcontent correctly"
 - [ ] ae64eaabdae serialize initial_connect_timeout as xcontent correctly
 - [ ] fa1a7c57b8e Add remote info to the HLRC (#49657)
-- [ ] cec6678587e Use peer recovery retention leases for indices without soft-deletes (#50351)
-- [ ] 3b8f5d9ea18 Modify proxy mode to support a single address (#50391)
-- [ ] 342a2920a96 Rename the remote connection mode simple to proxy (#50291)
-- [ ] 2d627ba7574 Add per-field metadata. (#49419)
-- [ ] 012746dd816 Send hostname in SNI header in simple remote mode (#50247)
-- [ ] ee92253f775 Adapt BWC after backporting #50214
-- [ ] 74ff50f814a Omit loading IndexMetaData when inspecting shards (#50214)
-- [ ] 7b863dc25b3 Recovery buffer size 16B smaller (#50100)
-- [ ] 972b81f8a9d Account trimAboveSeqNo in committed translog generation (#50205)
-- [ ] e0e7f1f89a4 Disk threshold decider is enabled by default (#50222)
-- [ ] 34f83904cc8 Adjust bwc for #48430
+- [x] cec6678587e Use peer recovery retention leases for indices without soft-deletes (#50351)
+- [s] 3b8f5d9ea18 Modify proxy mode to support a single address (#50391)
+- [s] 342a2920a96 Rename the remote connection mode simple to proxy (#50291)
+- [x] 2d627ba7574 Add per-field metadata. (#49419)
+- [s] 012746dd816 Send hostname in SNI header in simple remote mode (#50247)
+- [s] ee92253f775 Adapt BWC after backporting #50214
+- [x] 74ff50f814a Omit loading IndexMetaData when inspecting shards (#50214)
+- [s] 7b863dc25b3 Recovery buffer size 16B smaller (#50100)
+- [s] 972b81f8a9d Account trimAboveSeqNo in committed translog generation (#50205)
+- [s] e0e7f1f89a4 Disk threshold decider is enabled by default (#50222)
+- [s] 34f83904cc8 Adjust bwc for #48430
 - [x] b9fbc8dc748 Migrate peer recovery from translog to retention lease (#49448)
-- [ ] 0cedb9e2517 Update remote cluster stats to support simple mode (#49961)
+- [s] 0cedb9e2517 Update remote cluster stats to support simple mode (#49961)
 - [ ] 70af176dea3 Improve DateFieldMapper `ignore_malformed` handling (#50090)
```

Comment:
#50090 needs several prs as prerequisite to tackle it, therefore i leave it out for now.